gui/src/components/udx/cortx-udx-registration.vue

```diff
@@ -111,14 +111,22 @@
           </v-row>
 
           <v-row>
-            <v-col class="py-0 pr-0">
-              <br />
-              <h4>{{ $t("udx-registration.S3Account") }}: {{ registrationForm.accountName }}</h4>
-              <h4>{{ $t("udx-registration.IAMUser") }}: {{ registrationForm.iamUsername }}</h4>
-              <h4>{{ $t("udx-registration.bucketName") }}: {{ `ldp-${registrationForm.bucketName}` }}</h4>
-              <br />
+            <v-col class="p-0" md="4">
```

Comment:
not sure if this block required for defect?

Reply:
Removed